cfpb/sbl-frontend
# [Filing app] Phase 2 - Live data

Closes #286 

## Changes

- Pulls associated institutions from API
- Enables creation of a Filing from the UI (via the API)
- Displays current Filing status, as best we can determine, from API data

## How to test this PR

1. Log into the Frontend with a user that has at least one associated institution
2. Should see a status message indicating: `You have not started the Filing process`
3. Click `Start a Filing`
4. Should see a status message: `Upload your lending data`


## Screenshots
- Implemented statuses: `Start a Filing`, `Upload your data`
  <img width="1225" alt="Screenshot 2024-03-12 at 1 35 06 PM" src="https://github.com/cfpb/sbl-frontend/assets/2592907/94419ddd-9c9e-4291-9773-d20cd39c395a">
- Display when there's an error fetching data
  <img width="1167" alt="Screenshot 2024-03-12 at 1 14 06 PM" src="https://github.com/cfpb/sbl-frontend/assets/2592907/b12ac988-2725-48bd-9623-07f19d535281">
